src/components/shared/Menu/index.jsx

```diff
@@ -0,0 +1,58 @@
+import PropTypes from 'prop-types';
+import React, { useEffect, useState } from 'react';
+import { OuterSpace, StyledMenu } from './menu.styles';
+
+/**
+ * Renders Anchored Menu.
+ *
+ * @param {Object} anchorRef anchor element reference.
+ * @param {boolean} isOpen state of the menu.
+ * @param {children} children JSX.Element.
+ * @param {Function} onClose onClose handler.
+ * @param {boolean} [sticky=false] is anchor sticky.
```

Comment:
```suggestion
 * @param {Object}   anchorRef anchor element reference.
 * @param {boolean}  isOpen state of the menu.
 * @param {children} children JSX.Element.
 * @param {Function} onClose onClose handler.
 * @param {boolean}  [sticky=false] is anchor sticky.
```

Reply:
One more small thing here in addition to Mahmoud change, May you start the description of the parameters in capital letter and to modify these descritions little a bit, 
```suggestion
 * @param {Object}   anchorRef      Anchor element reference.
 * @param {boolean}  isOpen         Wheather the menu is opened or not.
 * @param {node}     children       Child nodes to render.
 * @param {Function} onClose        Close event handler.
 * @param {boolean}  [sticky=false] Wheather the anchor is sticky or not.
 ```

Reply:
done